plugins/services/src/js/components/forms/ServiceFormSection.js

```diff
@@ -0,0 +1,151 @@
+import React, {Component} from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FieldHelp from '../../../../../../src/js/components/form/FieldHelp';
+import FieldError from '../../../../../../src/js/components/form/FieldError';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+import FieldTextarea from '../../../../../../src/js/components/form/FieldTextarea';
+import Icon from '../../../../../../src/js/components/Icon';
+import ServiceConfigReducers from '../../reducers/ServiceConfigReducers';
+import ServiceValidationReducers from '../../reducers/ServiceValidationReducers';
+
+class ServiceFormSection extends Component {
+  getCMDLabel() {
+    let content = (
+      <span>
+        {"The command value will be wrapped by the underlying Mesos executor via /bin/sh -c ${cmd}. "}
+        <a href="https://mesosphere.github.io/marathon/docs/application-basics.html" target="_blank">
+          More information
+        </a>.
+      </span>
+    );
+
+    return (
+      <FieldLabel>
+        {"Command "}
+        <Tooltip
+          content={content}
+          interactive={true}
+          wrapText={true}
+          maxWidth={300}
+          scrollContainer=".gm-scroll-view">
+            <Icon color="grey" id="ring-question" size="mini" family="mini" />
+        </Tooltip>
+      </FieldLabel>
+    );
+  }
+
+  getIDHelpBlock() {
+    return (
+      <span>
+        {"Include the path to your service, if applicable. E.g. /dev/tools/my-service. "}
+        <a href="https://mesosphere.github.io/marathon/docs/application-groups.html" target="_blank">
+          More information
+        </a>.
+      </span>
+    );
+  }
+
+  render() {
+    let {data, errors} = this.props;
+
+    return (
+      <div className="form flush-bottom">
+        <div className="form-row-element">
+          <h2 className="form-header flush-top short-bottom">
+            Services
+          </h2>
+          <p>
+            Configure your service below. Start by giving your service a name.
+          </p>
+        </div>
+
+        <div className="flex row">
+          <FormGroup className="column-8" hasError={Boolean(errors.id)}>
+            <FieldLabel>
+              SERVICE NAME <span className="text-danger">*</span>
```

Comment:
#nit Let's use title casing for these labels; the CSS should enforce `text-transform: uppercase`. That way, if we choose not to display uppercase labels, we can just change it in the CSS.